openedx/core/djangoapps/cors_csrf/models.py

```diff
@@ -7,7 +7,7 @@
 class XDomainProxyConfiguration(ConfigurationModel):
```

Comment:
The table should remain as <app_name>_<model_name> all in lowercase. Only moving a model to a different app *or* changing the model name should affect the table name. However, it'd be worth running:
`python ./manage.py showmigrations --settings=devstack`
in your devstack to double-check.